recipes/cellprofiler/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "CellProfiler" %}
+{% set version = "3.1.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/CellProfiler/CellProfiler/archive/v{{ version }}.zip
+  sha256: 299d1be28df0b9c35e856177ada7555746d5b94a0f7af34dc3c3dd3671252a6a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  skip: True  # [osx]
+
+requirements:
+  host:
+    - pip
+    - python <3
+  run:
+    - python <3
+    - appdirs
+    - boto3
+    - cython
+    - h5py
+    - ipywidgets
+    - openjdk
+    - joblib
+    - jupyter
+    - libtiff
+    - libxml2
+    - libxslt
+    - lxml
+    - packaging
+    - pillow
+    - pip
+    - pyzmq=15.3.0
+    - mahotas
+    - matplotlib!=2.1.0,>2.0.0
+    - mysqlclient
+    - numpy
+    - raven
+    - requests
+    - scikit-image>=0.13
+    - scikit-learn
+    - scipy
+    - sphinx
+    - tifffile
+    - wxpython=3.0.2.0
+    - cellh5
+    - centrosome
+    - inflect
+    - prokaryote==2.4.0
+    - javabridge==1.0.15
```

Comment:
This is only available in version 1.0.14 afaik and this is also more then a year old.

Reply:
@bgruening Do you think we can get this package to work somehow or is it too cumbersome?

Reply:
You mean javabridge?

Reply:
yes, since we need it for python-bioformats